apps/guardian/src/shell/App/App.tsx

```diff
@@ -2,25 +2,34 @@ import { useAtomValue } from 'jotai';
 import { RouterProvider } from 'react-router-dom';
 import { clsx } from 'clsx';
 
-import { selectedAccountAtom, electionConfigAtom } from '~/shared/store';
+import { selectedAccountAtom, electionConfigAtom, connectionErrorAtom } from '~/shared/store';
 import { router } from '../router';
 import { PropsWithChildren, useMemo } from 'react';
 import { accountShowShort } from 'shared/util';
-import { AccountAddress } from '@concordium/web-sdk';
+import { BackendErrorType } from '~/shared/ffi';
+import { version } from '../../../package.json';
 
 type ConfigurationItemProps = PropsWithChildren<{
     className?: string;
     /** Whether the configuration item should render as "connected" */
     connected: boolean;
+    /** Whether the configuration item should signal an error */
+    error?: boolean;
```

Comment:
It's best practice to use booleans in components like this, as it allows you to _not_ specify a value for the property instead of having to declare `error={false}`. It enables writing JSX that is easier to read when the properties are used in a static way.

Reply:
Ok. Why is it not best practice to use a void here then? Since as far as I understand you really just need the presence of the field.

Reply:
I meant a unit type. I can see `void` is not actually that in typescript, so something like `[]` or a const.

I just meant that it seems like you want to represent two options, but the type you are using is Option<bool> which can represent three. Which annoys some people :P

Reply:
These are my questions to understand why, if it is standard practice then it is what it is.